### PR TITLE
[Cookbook][Security] some additional tweaks for the voter cookbook

### PR DESCRIPTION
**Pull request by xabbuh**
| Q | A |
| --- | --- |
| Doc fix? | yes |
| New docs? | no |
| Applies to | 2.6+ |
| Fixed tickets |  |

This applies the comments left by @WouterJ in e46f02e2fe8c0bec539f3736d7ee5ef371c2102c.

### CODE REVIEW

File: cookbook/security/voters_data_permission.rst [hunk 1]
```diff
@@ -104,8 +104,8 @@ edit a particular object. Here's an example implementation:
 
                     break;
                 case self::EDIT:
-                    // we assume that our data object has a method getOwner() to
-                    // get the current owner user entity for this data object
+                    // this assumes that the data object has a method getOwner()
```
Comment (javiereguiluz) [inline cookbook/security/voters_data_permission.rst]: `method getOwner()` -> `getOwner() method` ?

File: cookbook/security/voters_data_permission.rst [hunk 2]
```diff
@@ -104,8 +104,8 @@ edit a particular object. Here's an example implementation:
...
+                    // to get the current owner user entity for this data object
```
Comment (javiereguiluz) [inline cookbook/security/voters_data_permission.rst]: this is a bit difficult to read -> `the current owner user entity for this data object`

Comment (xabbuh) [inline cookbook/security/voters_data_permission.rst]: How would you write that sentence instead?

Comment (javiereguiluz) [inline cookbook/security/voters_data_permission.rst]: `to get the user entity for this data object owner`
`to get the user entity related to this object`
`to get the entity of the user who owns this data object`

Comment (xabbuh) [inline cookbook/security/voters_data_permission.rst]: good suggestions (I chose the third one)

**Comment by WouterJ** [comment]
Can you please rebase?

**Comment by xabbuh** [comment]
done

**Pull request merged**